src/refinement-crumbs/index.ts

```diff
@@ -61,7 +72,9 @@ namespace RefinementCrumbs {
 
   export interface State {
     label?: string;
+    navigationSelector?: (field: string) => any;
```

Comment:
Give the right return value type

Reply:
When I put `Store.Navigation[]` instead of `any` here it throws a bunch of errors when compiling

Reply:
That's because it's `Store.Navigation` not an array

Reply:
Sorry my bad, still throwing errors though. This is what I'm getting
```
Type '{ field: string; refinements: ({ index: number; field: string; range: boolean; selected: boolean;...' is not assignable to type 'State'.
  Object literal may only specify known properties, and 'field' does not exist in type 'State'.
```
I don't think creating a separate interface is the answer but I'm not sure what's going on.